r-package/grf/R/causal_forest.R

```diff
@@ -66,7 +66,7 @@
 #'                    to the maximum hardware concurrency.
 #' @param seed The seed of the C++ random number generator.
 #'
-#' @return A trained causal forest object.
+#' @return A trained causal forest object. Optional tuning output information is available in `tuning.output`.
```

Comment:
Small comment, maybe we could say 'If `tune.parameters` is enabled, then tuning information will be included through the `tuning.output` attribute.'